src/hotspot/share/runtime/threads.cpp

```diff
@@ -397,6 +397,8 @@ void Threads::initialize_java_lang_classes(JavaThread* main_thread, TRAPS) {
     JDK_Version::set_runtime_vendor_version(get_java_version_info(ik, vmSymbols::java_runtime_vendor_version_name()));
 
     JDK_Version::set_runtime_vendor_vm_bug_url(get_java_version_info(ik, vmSymbols::java_runtime_vendor_vm_bug_url_name()));
+
+    VM_Version::set_vm_vendor(get_java_version_info(ik, vmSymbols::java_runtime_vendor_vm_name()));
```

Comment:
Nit: not sure what naming convention should follow as VM_Version and JDK_Version no longer contain version info only.  

`vm_vendor` and `runtime_vendor_vm_bug_url` should be together.    Perhaps `JDK_Version` should be renamed to reflect that they are fields in `VersionProps` class and so `VM_Version::vm_vendor` can be moved/renamed to `JDK_Version::runtime_vendor_vm`.